src/modules/leaders.js

```diff
@@ -1,14 +1,20 @@
-class Leaders {
-    static scores = [];
+import getData from './generateLeaders.js';
+  const dataURL = "https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/6uj0uqnZpshKtXlAZwzJ/scores"; 
+    const getAPIData = () => {
+      fetch(dataURL)
+        .then((res) => res.json())
+        .then((data) => {
+          getData(data);
+        });
+    };
+    const postData = (data) => {
+      fetch(dataURL, {
+        method: "POST",
+        body: JSON.stringify(data),
+        headers: {
+          "Content-type": "application/json; charset=UTF-8",
+        },
+      });
+    };
```

Comment:
- [x] Great job so far 👍 your App displays well and works as expected but in the project requirements you are to use `async` and `await` JavaScript features to consume the API but you have implemented your features using `promises`. Kindly rectify this issue in all the cases where you are fetching data. Below is an implemented example:
- Async function:
![156390118-9761b0bc-b387-4542-b514-aafdfa7cc8aa](https://user-images.githubusercontent.com/85113567/156412606-818c17e0-595b-4bb4-a33b-69b7572037a5.jpg)